samples/nrf9160/modem_shell/src/curl/lib/curl_addrinfo.c

```diff
@@ -0,0 +1,600 @@
+/***************************************************************************
+ *                                  _   _ ____  _
+ *  Project                     ___| | | |  _ \| |
+ *                             / __| | | | |_) | |
+ *                            | (__| |_| |  _ <| |___
+ *                             \___|\___/|_| \_\_____|
+ *
+ * Copyright (C) 1998 - 2020, Daniel Stenberg, <daniel@haxx.se>, et al.
```

Comment:
I don't think we can have this license outside `/ext`

Reply:
Actually why is `cURL` under `/modem_shell` and not under `/ext` like `iperf` and `getopt`

Reply:
The reason is that curl is modified more than iperf3 and this curl version is further away from someone else using it in NRF. Decision was made finally by @carlescufi.
@rlubos and @carlescufi, please comment the license issue.

Reply:
It's derivative of MIT license.
Try to move to /ext and get rid of dependency from curl to mosh.

Reply:
Curl has been moved to /ext.